h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -15,13 +15,24 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
   events:
     '.annotator-toolbar mouseenter': 'show'
     '.annotator-toolbar mouseleave': 'hide'
-    'setTool': 'onSetTool'
     'setVisibleHighlights': 'onSetVisibleHighlights'
 
   html: '<div class="annotator-toolbar annotator-hide"></div>'
 
   pluginInit: ->
     @annotator.toolbar = @toolbar = $(@html)
+    # <UX-note>: If we detect a touch screen events, we use a different UX.
+    # This is because Mobile browsers often present their own popup menus in
+    # response to a user selection.
+    if 'ontouchstart' in window
+      # When there is a selection on touch devices show/hide highlighter button
+      # Check whether there is a user selection.
+      document.addEventListener "selectstart", =>
```

Comment:
You need to extract this into a function you can keep a reference to so that it can be unbound in a `detroy` method.